### PR TITLE
Add timestamps to the logs for downloader and stager

### PR DESCRIPTION
**Pull request by jogo**
Make it easier to understand how long things take from the logs, by
updating the default log format to include a timestamp. Note after this
change each env_name log will have more timestamps but many lines still
won't have timestamps if they are coming from the scripts (RESTARTING
etc.)

https://docs.python.org/3/library/logging.html#logrecord-attributes

### CODE REVIEW

**Review by robbintt — APPROVED** [review]
LGTM

**Comment by ankilosaurus** [comment]
Version bump before landing otherwise you will need another PR for it.

**Comment by jogo** [comment]
I am happy for this to wait until the next release and don't need a special release just for this.

**Pull request merged**
